### ISSUE TITLE
Plot adjustments 

### ISSUE THREAD

**Issue by GTsiliki** [ouzor/eyediagram#1]
Hello,

Is there a way to have a lanscape plot instead? 

And also I was wondering whether the nodes in the middle of the plot could slightly increase/or adjust to the labels inside?
Thanks

**Comment by ouzor** [comment]
Thanks for the comments. There is a lot of hard coded stuff right now, both
your requests would be possible but would need modifying the Processing
code. This has not been in active development for many years now, so I
don't even remember where to start. But feel free to try it out!

**Comment by GTsiliki** [comment]
True, thanks for the answer! I'll probably have a go, starting  with the R script!
